test/random/cuda/RngEngine.test.cu

```diff
@@ -152,3 +153,22 @@ TYPED_TEST(RngEngineFloatTest, generate_canonical)
 
     check_expected_float_samples(host_samples);
 }
+
+//---------------------------------------------------------------------------//
+// TEST on CPU
+//---------------------------------------------------------------------------//
+TEST(RngEngineCPUTest, generate_on_cpu)
+{
+  int num_samples = 1024*1000;
+  unsigned long seed = 12345u;
+
+  RngEngine rng(new RngState);
```

Comment:
This is a memory leak. For consistency with the other host tests, I suggest making a local `RngState host_rng_state;` and passing it in.

Reply:
Changed accordingly.